### PR TITLE
ADD: relevance API + new NER apis

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: R/relevance.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#' @export
+#' @import httr rjson stringr
+#' @examples
+#' result <- relevance("Barack Obama", "president")
+#' cat(sprintf("%s is mentioned in the input text" % result[0]['text']))
+#'
+relevance <- function(text, queries, api_key = FALSE, cloud = FALSE, version = NULL, ...) {
+
+  # Checks parameters
+  if (missing(text) || str_trim(text) == "") {
```
Comment (anneCarlson) [inline R/relevance.R]: Can we make this a decorator or whatever the equivalent is in R?

Comment (madisonmay) [inline R/relevance.R]: We could factor it out into a function, which is essentially the same thing.

Comment (madisonmay) [inline R/relevance.R]: But probably will address that one in another PR.

Comment (anneCarlson) [inline R/relevance.R]: Fair enough!

**Pull request merged**